site/docs/v1/tech/events-webhooks/event-bodies/user-registration-update-complete.adoc

```diff
@@ -0,0 +1,55 @@
+:type: user.registration.update.complete
+== User Registration Update Complete
+
+[NOTE.since]
+====
+This event has been available since 1.30.0
+====
+
+This event is generated when a user registration update transaction has completed.
```

Comment:
Maybe omit transaction here and describe it below in the Transaction Compatible section?